lib/ansible/modules/cloud/vmware/vmware_dvswitch_lacp.py

```diff
@@ -0,0 +1,402 @@
+                    lag_found = False
+                    for lag in self.link_aggregation_groups:
+                        result = self.get_lacp_lag_options(lag)
+                        if lacp_group.name == result[0]:
+                            lag_found = True
+                            break
+                    if lag_found is False:
+                        changed = changed_lags = True
+                        lacp_lag_list.append(
+                            self.create_lacp_group_spec('remove', lacp_group.key, lacp_group.name, None, None, None)
+                        )
+            else:
+                changed = changed_lags = True
+                lacp_lag_list = []
+                for lag in self.link_aggregation_groups:
+                    lag_name, lag_mode, lag_uplink_number, lag_load_balancing_mode = self.get_lacp_lag_options(lag)
+                    lacp_lag_list.append(
+                        self.create_lacp_group_spec(
+                            'add', None, lag_name, lag_uplink_number, lag_mode, lag_load_balancing_mode
+                        )
+                    )
+        else:
+            if self.dvs.config.lacpGroupConfig:
+                changed = changed_lags = True
+                lacp_lag_list = []
+                for lacp_group in self.dvs.config.lacpGroupConfig:
+                    lacp_lag_list.append(
+                        self.create_lacp_group_spec('remove', lacp_group.key, lacp_group.name, None, None, None)
+                    )
+        if changed_lags:
+            changed_list.append("link aggregation groups")
+            current_lags_list = []
+            for lacp_group in self.dvs.config.lacpGroupConfig:
+                temp_lag = dict()
+                temp_lag['name'] = lacp_group.name
+                temp_lag['uplink_number'] = lacp_group.uplinkNum
+                temp_lag['mode'] = lacp_group.mode
+                temp_lag['load_balancing_mode'] = lacp_group.loadbalanceAlgorithm
+                current_lags_list.append(temp_lag)
+            results['link_aggregation_groups_previous'] = current_lags_list
+
+        if changed:
+            if self.module.check_mode:
+                changed_suffix = ' would be changed'
+            else:
+                changed_suffix = ' changed'
+            if len(changed_list) > 2:
+                message = ', '.join(changed_list[:-1]) + ', and ' + str(changed_list[-1])
+            elif len(changed_list) == 2:
+                message = ' and '.join(changed_list)
+            elif len(changed_list) == 1:
+                message = changed_list[0]
+            message += changed_suffix
+            if not self.module.check_mode:
+                if changed_support_mode and self.support_mode == 'basic' and changed_lags:
+                    self.update_lacp_group_config(self.dvs, lacp_lag_list)
+                    # NOTE: You need to run the task again to change the support mode to 'basic' as well
```

Comment:
Could you please add this note in documentation as well ?

Reply:
sure